src/compile.rs

```diff
@@ -479,6 +478,145 @@ fn validate_module_contents<'a>(
+                                    "Invalid configuration option",
+                                    Some(file),
+                                    a.get_range(),
+                                    &format!(
+                                        "The valid options for {} are true and false",
+                                        constants::MONOLITHIC
+                                    ),
+                                ),
+                            ))
+                        }
+                    }
+                }
+                constants::HANDLE_UNKNOWN_PERMS => {
+                    if let Argument::Var(a) = &c.value {
+                        if a.as_ref() == "allow" || a.as_ref() == "deny" || a.as_ref() == "reject" {
+                            configs.insert(constants::HANDLE_UNKNOWN_PERMS.to_string(), c);
+                        } else {
+                            errors.append(CascadeErrors::from(
+                                ErrorItem::make_compile_or_internal_error(
+                                    "Invalid configuration option",
+                                    Some(file),
+                                    a.get_range(),
+                                    &format!(
+                                        "The valid options for {} are allow, deny, and reject",
+                                        constants::SYSTEM_TYPE
+                                    ),
+                                ),
+                            ))
+                        }
+                    }
+                }
+                _ => errors.append(CascadeErrors::from(
+                    ErrorItem::make_compile_or_internal_error(
+                        &format!("{} is not a supprted configuration", c.name.as_ref()),
+                        Some(file),
+                        c.name.get_range(),
+                        &format!(
+                            "The supported configurations are {}, {}, and {}",
+                            constants::SYSTEM_TYPE,
+                            constants::MONOLITHIC,
+                            constants::HANDLE_UNKNOWN_PERMS
+                        ),
+                    ),
+                )),
+            }
+        }
+        if !configs.contains_key(&constants::HANDLE_UNKNOWN_PERMS.to_string()) {
+            errors.append(CascadeErrors::from(
+                ErrorItem::make_compile_or_internal_error(
+                    &format!(
+                        "{} configuration must be included in the system",
+                        constants::HANDLE_UNKNOWN_PERMS
+                    ),
+                    Some(file),
+                    None,
+                    &format!(
+                        "Add a {} configuration to system {}",
+                        constants::HANDLE_UNKNOWN_PERMS,
+                        system.name
+                    ),
+                ),
+            ));
+        }
+
```

Comment:
What about multiple copies of the same config?

system foo {
let monolithic = true;
let monolithic = false;
}